app/controllers/favorite_tags_controller.rb

```diff
@@ -0,0 +1,46 @@
+class FavoriteTagsController < ApplicationController
+  skip_before_filter :store_location, only: [:create, :destroy]
+  before_filter :users_only
+  before_filter :load_user
+  before_filter :check_ownership
+
+  respond_to :html, :json
+
+  # POST /favorites_tags
+  def create
+    @favorite_tag = current_user.favorite_tags.build(params[:favorite_tag])
+    success_message = ts('You have successfully added %{tag_name} to your favorite tags.', tag_name: @favorite_tag.tag_name)
```

Comment:
Line is too long. [124/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.